new-lamassu-admin/src/pages/UserManagement/modals/CreateUserModal.js

```diff
@@ -74,7 +75,11 @@ const CreateUserModal = ({ state, dispatch }) => {
 
   const [createUser, { error }] = useMutation(CREATE_USER, {
     onCompleted: ({ createRegisterToken: token }) => {
-      setCreateUserURL(urlResolver(`/register?t=${token.token}`))
+      const queryParams =
+        process.env.REACT_APP_BUILD_TARGET === 'LAMASSU'
+          ? `t=${token.token}`
+          : `t=${token.token}&id=${base64.encode(usernameField)}`
```

Comment:
What does the ID does on the server side for this one? Might be a good idea to leave a comment on this